docs/azure-csi.md

```diff
@@ -14,9 +14,9 @@ If you want to deploy the Azure Disk storage class to provision volumes dynamica
 
 Before creating the instances you must first set the `azure_csi_` variables in the `group_vars/all.yml` file.
 
-All of the values can be retrieved using the azure cli tool which can be downloaded here: <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>
+All values can be retrieved using the azure cli tool which can be downloaded here: <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>
 
-After installation you have to run `az login` to get access to your account.
+After install you have to run `az login` to get access to your account.
```

Comment:
I am not sure this change..

Reply:
Here I understand that `install` is an act, not a noun

Reply:
This one is tricky, my english is not that good to say which one is the right one 😅

Reply:
Installation is correct. You could say "after the install [process]" or "after installing [the application]". "[to] install" is infinitive, which is not the correct tense here.